stat_handler.go

```diff
@@ -119,11 +134,32 @@ func (h *StatHandler) WorkerResumeHandler() func(http.ResponseWriter, *http.Requ
 	}
 }
 
+// ScoreBoardHandler returns http.HandlerFunc implementation for /scoreboard request
+func (h *StatHandler) ScoreBoardHandler() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if !reqMethodValidate(w, r, "GET") {
+			return
+		}
+
+		busy := len(h.Tracker.queueStack)
+
+		renderJSON(w, &ScoreBoardResponse{
+			TotalHandled:   int(h.Tracker.ScoreBoard.TotalSucceeded + h.Tracker.ScoreBoard.TotalFailed),
+			TotalSucceeded: int(h.Tracker.ScoreBoard.TotalSucceeded),
+			TotalFailed:    int(h.Tracker.ScoreBoard.TotalFailed),
+			MaxWorker:      h.Tracker.ScoreBoard.MaxWorker,
+			BusyWorker:     busy,
+			IdleWorker:     h.Tracker.ScoreBoard.MaxWorker - busy,
+		})
+	}
+}
+
 // BuildServeMux returns http.ServeMux object with registered endpoints
 func (h *StatHandler) BuildServeMux() *http.ServeMux {
 	mux := http.NewServeMux()
 
 	mux.HandleFunc("/stats", stats_api.Handler)
+	mux.HandleFunc("/scoreboard", h.ScoreBoardHandler())
```

Comment:
Can `ScoreBoard` data merge into `/worker/current` ?
Sorry, I don't know why `ScoreBoard` data should separate.
(plz tell me 🙏 )

Reply:
I think too.
bf6a75d merge `/scoreboard` and `/worker/current`.